setup.py

```diff
@@ -17,6 +17,7 @@
     "scikit-learn>=0.19.1",
     "scipy>=1.1.0",
     "seaborn>=0.9.0",
+    "mgcpy>=0.3.0"
```

Comment:
@j1c I am wondering if we should fix mgc == 0.3.0 for now because things may change with their interface, thoughts?

Reply:
At the moment mgcpy 0.3.0 doesn't return null distributions. Made a PR to address this but I will only get all the functionality I want from a newer version.